tests/typecheck/test/test_client.yml

```diff
@@ -35,3 +34,76 @@
       async_factory = AsyncRequestFactory()
       async_request = async_factory.get('foo')
       reveal_type(async_request)  # N: Revealed type is "django.core.handlers.asgi.ASGIRequest"
+-   case: client_follow_flag
+    main: |
+      from django.test.client import Client
+      client = Client()
```

Comment:
What's up with these test changes? Did you base this on some other branch that isn't `master`?

Reply:
@intgr whoops you are right, my bad 🤦  this also closed #1252. Will prepare a PR that reverts the changes shortly